lldb/source/Plugins/Process/Linux/NativeProcessLinux.cpp

```diff
@@ -219,7 +219,8 @@ static llvm::Error AddPtraceScopeNote(llvm::Error original_error) {
   Expected<int> ptrace_scope = GetPtraceScope();
   if (auto E = ptrace_scope.takeError()) {
     Log *log = GetLog(POSIXLog::Process);
-    LLDB_LOG(log, "error reading value of ptrace_scope: {0}", E);
+    LLDB_LOG(log, "error reading value of ptrace_scope: {0}",
```

Comment:
This seems like a great candidate for `LLDB_LOG_ERROR`

Reply:
Done.